app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -5,12 +5,21 @@ class HowToVerifyController < ApplicationController
     include RenderConditionConcern
 
     before_action :confirm_step_allowed
+    before_action :confirm_verify_info_step_needed
 
     check_or_render_not_found -> { self.class.enabled? }
 
     def show
+      if idv_session.skip_doc_auth == false
+        @selection = Idv::HowToVerifyForm::REMOTE
+      elsif idv_session.skip_doc_auth == true
+        @selection = Idv::HowToVerifyForm::IPP
+      else
+        @selection = nil
+      end
```

Comment:
if we wanted to DRY a bit we could assign from the if statement
```suggestion
      @selection = if idv_session.skip_doc_auth == false
         Idv::HowToVerifyForm::REMOTE
      elsif idv_session.skip_doc_auth == true
        Idv::HowToVerifyForm::IPP
      end
```
and then we can drop the `else` block entirely because the result of not matching either branch is that it evaluates to `nil`

Reply:
Done! I started a refactor on this with commit 655af8896bbb6588c055af04e5361dd75dda7234 and then saw your comment.  I cleaned it up even more with commit ba889ef828c16f1868bf666d92a83309d42bf2be
Thanks for the suggestion!!